recipe/meta.yaml

```diff
@@ -21,12 +21,12 @@ requirements:
     - pip
   run:
     - python >=3.6
+    - click
+    - coloredlogs
+    - entrypoints
     - jinja2
     - prettytable
     - simplejson
-    - coloredlogs
-    - ipython
```

Comment:
IPython is technically optional, but we don't give a fallback of just the standard Python REPL for some reason

Reply:
I saw this as well when I was comparing this recipe to the one we use for running tests. I think we should implement that fallback and I'm happy to add the dep back in for this tag if you think it's correct. I opted on the furthest reach of minimizing dependencies when I was looking at the recipe this morning.

Reply:
I think leaving it off is fine - we should fix it in happi.

Reply:
Ok, I will merge this then